AliveSFU/AliveSFU/ExerciseTileViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ExerciseTileViewController.swift
+//  AliveSFU
+//
+//  Created by Liam O'Shaughnessy on 2016-10-28.
+//  Copyright © 2016 SimonDevs. All rights reserved.
+//
+
+import UIKit
+
+class ExerciseTileViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = UIColor.black.withAlphaComponent(0.8)
+        self.showAnimate()
+        
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func closePopUp(_ sender: Any) {
+        self.removeAnimate()
+        //self.view.removeFromSuperview()
+    }
+    @IBAction func background1(_ sender: Any) {
+        self.removeAnimate()
+    }
+    @IBAction func background2(_ sender: Any) {
+        self.removeAnimate()
+    }
+    @IBAction func background3(_ sender: Any) {
+        self.removeAnimate()
+    }
+    @IBAction func background4(_ sender: Any) {
+        self.removeAnimate()
+    }
+    //when user touches outside of popover (red rectangle)
+    func showAnimate()
+    {
+        self.view.transform = CGAffineTransform(scaleX: 1.3, y: 1.3)
+        self.view.alpha = 0.0;
+        UIView.animate(withDuration: 0.25, animations: {
+            self.view.alpha = 1.0
+            self.view.transform = CGAffineTransform(scaleX: 1.0, y: 1.0)
+        });
+    } 
+    
+    
+    func removeAnimate()
+    {
+        UIView.animate(withDuration: 0.25, animations: {
+            self.view.transform = CGAffineTransform(scaleX: 1.3, y: 1.3)
+            self.view.alpha = 0.0;
+        }, completion:{(finished : Bool) in
+            if (finished)
+            {
+                self.view.removeFromSuperview()
+            }
+        });
+    }
```

Comment:
We don't need showAnimate and removeAnimate. Swift is handling all the animations. We might need the code later for other animations so just comment these out and don't delete em